test/unit/controllers/consolidations/DisassociateUcrControllerSpec.scala

```diff
@@ -37,20 +40,29 @@ import scala.concurrent.ExecutionContext.Implicits.global
 class DisassociateUcrControllerSpec extends ControllerLayerSpec with MockCache with ScalaFutures with OptionValues {
 
   private val page = mock[disassociate_ucr]
+  private val config = mock[AppConfig]
```

Comment:
why we would like to mock this ? Can we provide any fixed behaviour stub for action in tests ?